### DISCUSSION TITLE
Openroad compatibility with multi vt cells and mmmc

### DISCUSSION THREAD

**Discussion by faisaladilquadery** [The-OpenROAD-Project/OpenROAD#4062] [Q&A] [answered]
Can openroad work with multi VT cells ( HVT , LVT , SVT ) and swap between them in a single run ? 
also can openroad work with MultiModeMulti Corner File (mmmc) ?

**Comment by maliberty** [comment, marked as answer]
OR can swap between VT cells.  OR supports multi-corner but not multi-mode STA.

**Comment by rovinski** [comment]
> can openroad work with MultiModeMulti Corner File (mmmc) ?

If you are asking about MMMC files from commercial vendors, those are proprietary formats which we don't support. We support multi-corner analysis as mentioned. You can look in the [OpenSTA manual](https://github.com/The-OpenROAD-Project/OpenSTA/blob/master/doc/OpenSTA.pdf) to find the relevant commands.
